dev-deployment.sh

```diff
@@ -31,9 +32,22 @@ cfLogin() {
   fi
 }
 
+cfSetDatabaseConnectionString() {
+    cf unset-env $APP_NAME DATABASE_CONNECTION_STRING
+    DATABASE_URL="$(cf env $APP_NAME | grep -o '"postgres://[^"]*' | tr -d '"')"
+    cf set-env $APP_NAME DATABASE_CONNECTION_STRING "$DATABASE_URL?ssl=true"
+}
+
+cfBindWithDatabase() {
+    cf bind-service $APP_NAME verify-local-matching-service-example-db
+    cf restart $APP_NAME
+}
+
 ./docker-pre-commit.sh
 
-cfLogin
+#cfLogin
```

Comment:
This should presumably not be commented out?

Reply:
oh yes, good catch.